mjpc/tasks/humanoid/tracking/task.xml

```diff
@@ -0,0 +1,164 @@
+
+    <framepos name="tracking_pos[pelvis]" objtype="site" objname="tracking[pelvis]" />
+    <framepos name="tracking_pos[head]" objtype="site" objname="tracking[head]" />
+    <framepos name="tracking_pos[ltoe]" objtype="site" objname="tracking[ltoe]" />
+    <framepos name="tracking_pos[rtoe]" objtype="site" objname="tracking[rtoe]" />
+    <framepos name="tracking_pos[lheel]" objtype="site" objname="tracking[lheel]" />
+    <framepos name="tracking_pos[rheel]" objtype="site" objname="tracking[rheel]" />
+    <framepos name="tracking_pos[lknee]" objtype="site" objname="tracking[lknee]" />
+    <framepos name="tracking_pos[rknee]" objtype="site" objname="tracking[rknee]" />
+    <framepos name="tracking_pos[lhand]" objtype="site" objname="tracking[lhand]" />
+    <framepos name="tracking_pos[rhand]" objtype="site" objname="tracking[rhand]" />
+    <framepos name="tracking_pos[lelbow]" objtype="site" objname="tracking[lelbow]" />
+    <framepos name="tracking_pos[relbow]" objtype="site" objname="tracking[relbow]" />
+    <framepos name="tracking_pos[lshoulder]" objtype="site" objname="tracking[lshoulder]" />
+    <framepos name="tracking_pos[rshoulder]" objtype="site" objname="tracking[rshoulder]" />
+    <framepos name="tracking_pos[lhip]" objtype="site" objname="tracking[lhip]" />
+    <framepos name="tracking_pos[rhip]" objtype="site" objname="tracking[rhip]" />
+
+    <framelinvel name="tracking_linvel[pelvis]" objtype="site" objname="tracking[pelvis]" />
+    <framelinvel name="tracking_linvel[head]" objtype="site" objname="tracking[head]" />
+    <framelinvel name="tracking_linvel[ltoe]" objtype="site" objname="tracking[ltoe]" />
+    <framelinvel name="tracking_linvel[rtoe]" objtype="site" objname="tracking[rtoe]" />
+    <framelinvel name="tracking_linvel[lheel]" objtype="site" objname="tracking[lheel]" />
+    <framelinvel name="tracking_linvel[rheel]" objtype="site" objname="tracking[rheel]" />
+    <framelinvel name="tracking_linvel[lknee]" objtype="site" objname="tracking[lknee]" />
+    <framelinvel name="tracking_linvel[rknee]" objtype="site" objname="tracking[rknee]" />
+    <framelinvel name="tracking_linvel[lhand]" objtype="site" objname="tracking[lhand]" />
+    <framelinvel name="tracking_linvel[rhand]" objtype="site" objname="tracking[rhand]" />
+    <framelinvel name="tracking_linvel[lelbow]" objtype="site" objname="tracking[lelbow]" />
+    <framelinvel name="tracking_linvel[relbow]" objtype="site" objname="tracking[relbow]" />
+    <framelinvel name="tracking_linvel[lshoulder]" objtype="site" objname="tracking[lshoulder]" />
+    <framelinvel name="tracking_linvel[rshoulder]" objtype="site" objname="tracking[rshoulder]" />
+    <framelinvel name="tracking_linvel[lhip]" objtype="site" objname="tracking[lhip]" />
+    <framelinvel name="tracking_linvel[rhip]" objtype="site" objname="tracking[rhip]" />
+  </sensor>
+
+  <include file="./keyframes/CMU-CMU-02-02_04_poses.xml" />  <!-- simple jump, balance -->
```

Comment:
It would be really nice to have the keyframes loadable from command line with their filenames. I haven't yet looked at how to do that though, so if anyone has ideas, let me know.

Reply:
@nimrod-gileadi is there an easy way to do this?

Reply:
Not super easy. You'll need to create the MuJoCo XML in code, which I guess you can hack to do.

Once you have an XML string, you can load it as a model with code like this: https://github.com/deepmind/mujoco/blob/main/test/fixture.cc#L71